lib/screens/map_view.dart

```diff
@@ -85,18 +88,43 @@ class _MapViewScreenState extends State<MapViewScreen> {
           MarkerLayerOptions(
             markers: [
               Marker(
-                  width: 40.0,
-                  height: 40.0,
+                  width: 110.0,
+                  height: 110.0,
                   point: latlng.LatLng(locationData?.latitude ?? 51.5,
                       locationData?.longitude ?? -0.09),
-                  builder: (ctx) => const Beastie())
+                  builder: (ctx) => const Avatar()),
+              spawnBeastie(),
+              spawnBeastie(),
+              spawnBeastie()
             ],
           ),
         ],
       );
     }
   }
 
+  Marker spawnBeastie() {
```

Comment:
It's unclear to me why we would need both a Beastie and spawnBeastie class. I think we should only keep one and specifically, it would  be good if it was actually a class so that we don't re-render it unnecessarily.